contracts/multisig/src/msg.rs

```diff
@@ -56,12 +57,12 @@ pub enum ExecuteMsg {
         to sign their own address using the private key */
         signed_sender_address: HexBinary,
     },
-    // Authorizes a contract to call StartSigningSession. Callable only by governance
-    #[permission(Governance)]
-    AuthorizeCaller { contract_address: Addr },
-    // Unauthorizes a contract, so it can no longer call StartSigningSession. Callable only by governance
+    // Authorizes a set of contracts to call StartSigningSession.
     #[permission(Governance)]
-    UnauthorizeCaller { contract_address: Addr },
+    AuthorizeCallers { contracts: Vec<(String, ChainName)> },
+    // Unauthorizes a set of contracts, so they can no longer call StartSigningSession.
+    #[permission(Elevated)]
+    UnauthorizeCallers { contracts: Vec<(String, ChainName)> },
```

Comment:
make these hashmaps instead? That means we don't have to deal with duplicates and non-rust users will have a bad time constructing these tuples again